components/raftstore/src/store/peer.rs

```diff
@@ -806,6 +806,7 @@ where
     /// The index of last scheduled committed raft log.
     pub last_applying_idx: u64,
     pub max_apply_unpersisted_log_limit: u64,
+    enable_apply_unpersisted_log_state: bool,
```

Comment:
nit: is it necessary to add this member solely for triaging the trigger types of the `enable` and `disable` actions related to the "apply unpersisted log" feature?

If necessary, please add some annotations here.

Reply:
Done